README.md

```diff
@@ -172,9 +172,13 @@ start the devcontainer and use `file-find` `/docker:user@container:/path/to/file
 more detail please refer to [devcontainer-feature-emacs-lsp-bridge](https://github.com/nohzafk/devcontainer-feature-emacs-lsp-bridge).
 
 If you use `apheleia` as formatter, `lsp-bridge` now support auto formatting file on devcontainer.
+```elisp
+;; setup PATH for remote command execution
+(with-eval-after-load 'tramp
+  (add-to-list 'tramp-remote-path "~/.nix-profile/bin")
```

Comment:
lsp-bridge user must be install Nix?

Reply:
container support is done by using the devcontainer feature which use nix to install lsp-bridge and language server.

I'm planing to create a separate repo for instructions and setup of how to use lsp-bridge in container. Will update README to point to the new repo later.